MYOB.API.SDK/SDK/SDK.csproj

```diff
@@ -259,7 +259,7 @@
     <Compile Include="Contracts\Version2\Contact\CustomerLink.cs" />
     <Compile Include="Contracts\Version2\Contact\CustomerSellingDetails.cs" />
     <Compile Include="Contracts\Version2\Contact\EmployeeLink.cs" />
-	  <Compile Include="Contracts\Version2\Contact\ContactLink.cs" />
+	<Compile Include="Contracts\Version2\Contact\ContactLink.cs" />
```

Comment:
Tabbing issue?

Reply:
Yes, it is and this will fix that tabbing issue